ai_edge_torch/generative/examples/stable_diffusion/decoder.py

```diff
@@ -22,129 +22,131 @@
 import ai_edge_torch.generative.layers.unet.model_config as unet_cfg
 import ai_edge_torch.generative.utilities.stable_diffusion_loader as stable_diffusion_loader
 
-TENSORS_NAMES = stable_diffusion_loader.AutoEncoderModelLoader.TensorNames(
-    post_quant_conv="0",
-    conv_in="1",
+TENSOR_NAMES = stable_diffusion_loader.AutoEncoderModelLoader.TensorNames(
```

Comment:
just curious, will this mapping change if we are using a different SD 1.5 checkpoint?

Reply:
The tensor names are defined by the model implementation. As long as it use the same model implementation, in this case, `StableDiffusionPipeline` in `diffuser` from HuggingFace, the mapping would be the same.